README.md

```diff
@@ -40,7 +40,7 @@ pip install -i https://test.pypi.org/simple/ eda_analysis
 
```

Comment:
Are you able to remove the bolded part from the description of NA values:

3. `describe_na_values` : This function will take in dataframe and will **plot heat map to locate NA values in each feature and will also** give a table listing number of NA values in each feature.

Reply:
Im also making this change on the R package

Reply:
Oh yes, we need to remove that.

Reply:
Removed.